cli/src/command.rs

```diff
@@ -415,6 +420,17 @@ pub fn run() -> Result<()> {
 			#[cfg(not(feature = "polkadot-native"))]
 			panic!("No runtime feature (polkadot, kusama, westend, rococo) is enabled")
 		},
+		Some(Subcommand::HostPerfCheck(cmd)) => {
+			let mut builder = sc_cli::LoggerBuilder::new("info");
```

Comment:
```suggestion
			let mut builder = sc_cli::LoggerBuilder::new("");
```

We always log `info` by default.